src/main/java/org/apache/maven/plugins/war/util/PathSet.java

```diff
@@ -53,14 +52,14 @@ static String normalizeSubPath( String path )
         {
             return path;
         }
-        String cleanPath = path.replaceAll( "[\\\\]+", SEPARATOR )
-                .replaceAll( "[/]+" , SEPARATOR );
-        cleanPath = cleanPath.charAt( 0 ) == '/' ? cleanPath.substring( 1 ) : cleanPath;
+        String cleanPath = path.replaceAll( "[\\\\]+", File.separator )
+                .replaceAll( "[/]+" , File.separator );
+        cleanPath = cleanPath.charAt( 0 ) == File.separatorChar ? cleanPath.substring( 1 ) : cleanPath;
         if ( cleanPath.isEmpty() )
         {
             return cleanPath;
         }
-        if ( cleanPath.charAt( cleanPath.length() - 1 ) == '/' )
+        if ( cleanPath.charAt( cleanPath.length() - 1 ) == File.separatorChar )
```

Comment:
so this will be '/' (and we will introduce a constant)

Reply:
Ops, my last commit will break the code on Windows. File.separatorChar on Windows is '\', not '/'.

Reply:
Don't worry.
We will test your commit over all supported versions of Maven, Java and Linux and Windows.
Hopefully if you are breaking things we will notice

Reply:
Since I reverted the last modifications due to broken code, would you have other remarks on the code?

Reply:
@eolivelli I will do as you said. Please, give me some minutes.